apps/pie-docs/test/system/footer.spec.js

```diff
@@ -12,13 +12,16 @@ test.describe('PIE - Footer - @desktop', () => {
         for await (const link of await page.getByTestId('footer_link').all()) {
             const href = await link.getAttribute('href');
 
-            await Promise.all([
-                page.waitForResponse((resp) => resp.url().includes(href))
-                .then((resp) => {
-                    expect(resp.status(), `${resp.url()} should return successful status code`).toBe(200);
-                }),
-                link.click()
-            ]);
+            /* privacy-policy is known to be flaky */
```

Comment:
I think this warrants a changeset so we can easily look back on this if needed

Reply:
added `TODO: DSW-2047 - Replace footer link test with a snapshot test`
[DSW-2047](https://justeattakeaway.atlassian.net/browse/DSW-2047)